python/across_api/base/database.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+import arc  # type: ignore[import]
+import boto3  # type: ignore[import]
+import os
+
+
+def dynamodb_table(tablename) -> Any:
+    """If running in Architect, return tables.table, else return boto3 dynamodb
+    table. This enables the use of moto to mock the dynamodb table in tests."""
+    if os.environ.get("ARC_ENV") is not None:
+        return arc.tables.table(tablename)
+    else:
+        session = boto3.Session()
+        return session.resource("dynamodb", region_name="us-east-1").Table(tablename)
```

Comment:
Remove; you only added this to support mock testing.

Reply:
It's gone